src/prerender.ts

```diff
@@ -257,7 +257,9 @@ function extractLinks(
 
   // Extract from x-nitro-prerender headers
   const header = res.headers.get("x-nitro-prerender") || "";
-  _links.push(...header.split(",").map((i) => i.trim()));
+  _links.push(...header.split(",")
+      .map((i) => decodeURIComponent(i))
+      .map((i) => i.trim()));
```

Comment:
I think we should move trim before decode

Reply:
Yes, good catch. I've updated the PR.